### PR TITLE
Time-series charts and card time ranges

### PR DESCRIPTION
**Pull request by bryancboyd**
**Summary**

- Initial drop of time series cards with carbon charts
- Added time range dropdown for cards

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     }
   },
   "dependencies": {
+    "@carbon/charts-react": "0.12.1",
```
Comment (scottdickerson) [inline package.json]: we need to remove this dependency and only have it in a peer dep I think...

File: src/components/Card/Card.jsx [hunk 2]
```diff
@@ -220,8 +238,7 @@ const Card = ({
     </StyledToolbar>
   ) : (
     <StyledToolbar>
-      {// TODO: if we keep this, expose capability under prop
-      false && timeBoxSelection}
+      {mergedAvailableActions.range && timeBoxSelection}
```
Comment (scottdickerson) [inline src/components/Card/Card.jsx]: I think we should also hide the range picker if the Card is in the editable state for now...

File: src/components/TimeSeriesCard/TimeSeriesCard.jsx [hunk 3]
```diff
@@ -1,438 +1,142 @@
...
+
+  const sameYear =
+    moment(moment.unix(values[0].timestamp / 1000)).isSame(moment(), 'year') &&
+    moment(moment.unix(values[values.length - 1].timestamp / 1000)).isSame(moment(), 'year');
+
+  const formatInterval = (timestamp, index) => {
+    const m = moment.unix(timestamp / 1000);
+
+    return sameDay && interval === 'hour' && index === 0
+      ? m.format('DD MMM YYYY')
```
Comment (scottdickerson) [inline src/components/TimeSeriesCard/TimeSeriesCard.jsx]: are we tracking the localization work somewhere?

File: src/components/Card/Card.jsx (hunk 2 above)
Comment (bryancboyd) [reply src/components/Card/Card.jsx]: yep, that's covered by the ternary branch on `isEditable` -- if it's in editable state the range picker (and expansion button) won't be rendered

File: package.json (hunk 1 above)
Comment (bryancboyd) [reply package.json]: good call thanks

File: src/components/TimeSeriesCard/TimeSeriesCard.jsx (hunk 3 above)
Comment (bryancboyd) [reply src/components/TimeSeriesCard/TimeSeriesCard.jsx]: not at the moment

**Pull request merged**